lib/project_types/theme/messages/messages.rb

```diff
@@ -21,16 +21,35 @@ module Messages
           },
         },
         forms: {
+          ask_password: "Password:",
+          ask_store: "Store domain:",
           create: {
-            ask_password: "Password:",
-            ask_store: "Store domain:",
             ask_title: "Title:",
-            errors: "%s can't be blank",
             private_app: <<~APP,
               To create a new theme, Shopify App CLI needs to connect with a private app installed on your store. Visit {{underline:%s/admin/apps/private}} to create a new API key and password, or retrieve an existing password.
               If you create a new private app, ensure that it has Read and Write Theme access.",
             APP
           },
+          errors: "%s can't be blank",
+          pull: {
+            private_app: <<~APP,
+              To fetch your existing themes, Shopify App CLI needs to connect with your store. Visit {{underline:%s/admin/apps/private}} to create a new API key and password, or retrieve any existing password.
+              If you create a new private app, ensure that it has Read and Write Theme access.",
+            APP
+          },
+        },
+        pull: {
+          help: <<~HELP,
+            {{command:%s pull}}: Connects an existing theme in your store to Shopify App CLI. Creates a local copy of theme files on Shopify.
```

Comment:
```suggestion
            {{command:%s pull}}: Connects an existing theme in your store to Shopify App CLI. Downloads a copy of the theme files to your local development environment.
```

Is this an accurate description of what happens?